### PR TITLE
Amzn2.0 update

### PR DESCRIPTION
**Pull request by smithers1221**

### CODE REVIEW

File: install_scripts/templates/common/system.sh [hunk 1]
```diff
@@ -115,8 +115,8 @@ detectLsbDist() {
                 [ $1 -ge 6 ] && LSB_DIST=$_dist && DIST_VERSION=$_version && DIST_VERSION_MAJOR=$1
                 ;;
             amzn)
-                _error_msg="$_error_msg\nHowever detected version $_version is not one of 2017.09, 2017.03, 2016.09, 2016.03, 2015.09, 2015.03, 2014.09, 2014.03."
-                [ "$_version" = "2017.03" ] || [ "$_version" = "2017.09" ] || \
+                _error_msg="$_error_msg\nHowever detected version $_version is not one of 2.0, 2017.09.1, 2017.09, 2017.03, 2016.09, 2016.03, 2015.09, 2015.03, 2014.09, 2014.03."
```
Comment (laverya) [inline install_scripts/templates/common/system.sh]: Do we need `2017.09.1`? We already had `2017.09`

File: install_scripts/templates/common/system.sh [hunk 2]
```diff
@@ -115,8 +115,8 @@ detectLsbDist() {
...
+                [ "$_version" = "2017.03" ] || [ "$_version" = "2017.09" ] || [ "$_version" = "2.0" ] || \
```
Comment (laverya) [inline install_scripts/templates/common/system.sh]: Perhaps have the 2.0 check start a new line above this? For consistency

File: install_scripts/templates/common/docker-install.sh [hunk 3]
```diff
@@ -133,7 +133,7 @@ _installDocker() {
         compareDockerVersions "17.0.0" "${1}"
```
Comment (laverya) [inline install_scripts/templates/common/docker-install.sh]: Add 17.06 to the list here or edit 17.03

File: install_scripts/templates/common/system.sh [hunk 4]
```diff
@@ -115,8 +115,9 @@ detectLsbDist() {
                 [ $1 -ge 6 ] && LSB_DIST=$_dist && DIST_VERSION=$_version && DIST_VERSION_MAJOR=$1
                 ;;
             amzn)
-                _error_msg="$_error_msg\nHowever detected version $_version is not one of 2017.09, 2017.03, 2016.09, 2016.03, 2015.09, 2015.03, 2014.09, 2014.03."
-                [ "$_version" = "2017.03" ] || [ "$_version" = "2017.09" ] || \
+                _error_msg="$_error_msg\nHowever detected version $_version is not one of 2.0, 2017.09, 2017.03, 2016.09, 2016.03, 2015.09, 2015.03, 2014.09, 2014.03."
+                [ "$_version" = "2.0" ] || \
+		[ "$_version" = "2017.03" ] || [ "$_version" = "2017.09" ] || \
```
Comment (emosbaugh) [inline install_scripts/templates/common/system.sh]: Indentation is off here

**Comment by smithers1221** [comment]
@emosbaugh - thanks for catching and the tip - twice it appeared 'normal' in the editor and the indent error was only visible on the github diff.

**Comment by smithers1221** [comment]
@emosbaugh - oh, also looks like I need write access for this repo in order to merge PR...

**Pull request merged**